env/env.js

```diff
@@ -33,43 +31,43 @@ const base = removeNullValues({
   ...statePrimary,
   ALLOY_API_KEY: ALLOY_SANDBOX_API_KEY,
   ALLOY_API_SECRET: ALLOY_SANDBOX_API_SECRET,
+  SENDINBLUE_API_KEY,
   MG_API_KEY,
   TWILIO_SID,
   TWILIO_TOKEN,
   TWILIO_FAX_NUMBER,
   RECEIVE_FAX_NUMBER,
-  SENDINBLUE_API_KEY,   // We are using the v3 API
-  SENDINBLUE_LIST_ID: 4,  // This is just a fake testing list -- no emails will be sent
-  USER_MAX_ORGS: 15,
+  USER_MAX_ORGS: 8,
   REACT_APP_BRAND_NAME: 'VoteByMail.io',
   REACT_APP_URL: 'https://votebymail.io/',
   REACT_APP_ELECTION_AND_DATE: 'the General Election on Tuesday, November 3rd, 2020',
-  REACT_APP_FEEDBACK_ADDR: 'feedback@votebymail.io',
-  REACT_APP_ELECTION_OFFICIAL_ADDR: 'elections@votebymail.io',
-  REACT_APP_SUPPORT_ADDR: 'support@votebymail.io',
-  MG_DOMAIN: 'email.votebymail.io',
-  MG_FROM_ADDR: 'Vote by Mail Application <application@email.votebymail.io>',
-  MG_REPLY_TO_ADDR: 'Vote by Mail Application <application@votebymail.io>',
+  REACT_APP_FEEDBACK_ADDR: 'feedback@mailmyballot.org',
```

Comment:
I think you're merging in the wrong branch.  This seems old and `SENDINBLUE_API_KEY` has been in the codebase for some time.